### ISSUE TITLE
Add better support for Razor syntax highlighting

### ISSUE THREAD

**Issue by Matthew-Bonner** [aspnet/Docs#7172]
Highlight js currently has aspx-csharp and cshtml.

I'm not sure what the difference is between the two but neither highlight Razor syntax very well.

This is what I get in Visual Studio:

![image](https://user-images.githubusercontent.com/7858744/41656469-fca980c0-7487-11e8-8285-b3d033a05960.png)

This is what I get in the docs:
![image](https://user-images.githubusercontent.com/7858744/41656483-084fd492-7488-11e8-81be-bccf2b7156ca.png)

**Comment by guardrex** [comment]
@vicancy Is this a DocFX issue, or is it an implementation issue on the repo side of things?

**Comment by scottaddie** [comment]
The `aspx-csharp` option sounds like it supports Web Forms syntax highlighting. `cshtml` is definitely what we want to use.

**Comment by adkinn** [comment]
@scottaddie yes, we're supporting Web Forms right now. Adding cshtml highlighting is in our backlog.
